examples/scenes/src/pico_svg.rs

```diff
@@ -288,3 +282,38 @@ fn modify_opacity(color: Color, attr_name: &str, node: Node<'_, '_>) -> Color {
         color
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::parse_color;
+    use vello::peniko::color::{palette, AlphaColor, Srgb};
+
+    fn assert_close_color(c1: AlphaColor<Srgb>, c2: AlphaColor<Srgb>) {
+        const EPSILON: f32 = 1e-4;
+        assert_eq!(c1.cs, c2.cs);
+        for i in 0..4 {
+            assert!((c1.components[i] - c2.components[i]).abs() < EPSILON);
+        }
+    }
+
+    #[test]
+    fn parse_colors() {
```

Comment:
Hmm. Is it best practise to have tests which are only testing your dependencies? More tests are always good, I suppose.

The thing which gives me pause is that if this stops working, there isn't any remediation possible in this repository. But OTOH, this test failing would still be a good thing, as it's testing an assumption we're making, and would catch that before "shipping" it.

Reply:
Partly it is a signal that this works since Raph asked and I wasn't fully sure.